js/contacts.js

```diff
@@ -47,52 +51,15 @@ $.ajax({
                         </div>`;                  
                 var element = document.getElementById("test");
                 element.innerHTML = result;
-                $('#test_error').show();
-
+                $('#test').show();
             }
         } 
     }
+    
 }
 }).fail(function (response, code) {
     console.log(response)
     //alert('The token is invalid')
-});
-document.getElementById('elem').onclick = function () { 
-    console.log('clicked');
-    var parts = document.getElementById('token').value.split('@');
-    var token = parts[0];
-    var providerDomain = parts[1];
-
-    var data = 'providerDomain=' + encodeURIComponent(providerDomain) +
-  '&token=' + encodeURIComponent(token);
 
-    var baseUrl = OC.generateUrl('/apps/sciencemesh');
-    $.ajax({
-        url: baseUrl + '/contacts/accept',
-        type: 'POST',
-        contentType: 'application/x-www-form-urlencoded',
-        data: data
-    }).done(function (response) {
-      
-       if(response === '' || response === false) {
-            var element = document.getElementById("test_error");
-            element.innerHTML= 'No Sciencemesh Connection';
-        } else {
-            let result = JSON.parse(response);
-            if(result.hasOwnProperty('message')) {
-                let test = result.message
-                var element = document.getElementById("test_error");
-                element.innerHTML=test;
-
-                $('#provider').hide();
-                $('#display_name').hide();
-            } else {
-                console.log(result)
-            }
-        }
-     
-    }).fail(function (response, code) {
-        console.log(response)
-        //alert('The token is invalid')
-    });
-};
```

Comment:
Why remove this code block? Is it moved elsewhere?
These changes also seem unrelated to "add federated share type to sciencemeshShareProviderFactory"?